spec/services/reports/iaa_billing_report_spec.rb

```diff
@@ -156,6 +157,8 @@
     MonthlySpAuthCount.create(issuer: issuer2, ial: 2, year_month: today_year_month, user_id: 3,
                               auth_count: 3)
 
-    expect(subject.call).to eq(results_for_2_iaas.to_json)
```

Comment:
You could use `match_array` for an order-independent comparison?

```
    expect(JSON.parse(subject.call)).to match_a(results_for_2_iaas)
```

Reply:
NICE!

Reply:
TIL 😮

Reply:
didn't work because I have to also symbolize keys (or stringify keys).  I'm just going to merge... but thanks for the cool snippet.